docs/admin/access.rst

```diff
@@ -87,65 +87,70 @@ The default value can be changed by :setting:`DEFAULT_ACCESS_CONTROL`.
 Managing per-project access control
 -----------------------------------
 
-Users with the :guilabel:`Manage project access` privilege (see
-:ref:`privileges`) can manage users in projects via adding them to the teams.
-The initial collection of teams is provided by Weblate, but additional ones can
-be defined providing more fine-grained access control. You can limit teams to
-languages and assign them designated access roles (see :ref:`privileges`).
+For `Public`, `Protected` and `Private` projects:
 
-The following teams are automatically created for every project:
+Granting users :guilabel:`Manage project access` (see :ref:`privileges`)
```

Comment:
I would argue it doesn't need to be pointed out that users need to have MPA to do what is explained to be recursive about MPA.
My original intent must have been to replace the duplicated text, and then I didn't remove the original one.
Maybe it makes sense now, but not at all sure.

Some of the text is moved to a hint below.